src/features/commands.ts

```diff
@@ -1141,6 +1141,60 @@ _ _
       });
     },
   },
+  {
+    words: ["!auth", "!authentication"],
+    help: "Provides a list of popular backend- and meta-frameworks",
+    category: "Web",
+    handleMessage: (msg) => {
+      const firstMention = msg.mentions.users.first();
+
+      const embed = {
+        title: "",
+        type: EmbedType.Rich,
+        description: `
+Authentication is a critical part of most web applications. Below are some resources to help you get started with authentication:
```

Comment:
I'd love to also mention something about JWTs here, whatcha think? Putting it here vs in the 'fields' in part because titles are longer, and to call it out as more opinion than resource

```suggestion
Authentication is a critical part of most web applications. Here are some resources to help you get started.
- [JSON Web Tokens (JWT) are Dangerous for User Sessions](https://redis.io/blog/json-web-tokens-jwt-are-dangerous-for-user-sessions/)
- [JWT should not be your default for sessions](https://evertpot.com/jwt-is-a-bad-default/)
```

Reply:
Agree 👍🏼